### PR TITLE
Fixed the visual bug caused by using Bootswatch skins

### PR DESCRIPTION
**Pull request by mrgame64**
Fixes the #84 issue

### CODE REVIEW

File: templates/modules/taskbar.tpl [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<div id="taskbar" class="taskbar navbar navbar-default navbar-fixed-bottom">
```
Comment (julianlam) [inline templates/modules/taskbar.tpl]: Right, this makes sense as the taskbar is transparent in Persona :+1:  (Note to self)

File: modules/taskbar.less [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	display: none;
 	z-index: 10001;
 	width: 50%;
-	left: 50%;
```
Comment (julianlam) [inline modules/taskbar.less]: What is the use of this `!important` declaration? This always raises a red flag for me, and testing on my dev shows that the `left` is already `50%`

**Comment by julianlam** [comment]
dd2a96a975b1cd4c7aeae3e6c7b6e011d05d71de Merged in with changes

v3.0.8

File: modules/taskbar.less (hunk 2 above)
Comment (mrgame64) [inline modules/taskbar.less]: For some reason when I was testing it, the Chrome inspector showed `left` being set to `0`.
As I noticed it originally was set to `50%`, but later overridden by something else, I wanted to make sure that it's forced to be `50%`

Comment (julianlam) [inline modules/taskbar.less]: Seemed weird that it had 50% width to begin with, so now it's back to 100%

Comment (mrgame64) [inline modules/taskbar.less]: Seems like I haven't given it much thought, then...
f8e45cc does indeed make more sense

**Pull request closed without merging**